sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/options/ListTablesOptions.java

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.data.tables.models;
+package com.azure.data.tables.options;
```

Comment:
I know our guidelines suggest using `options` package but we haven't enforced that and all other services have used `models` package to create options types. So, we should just put this back in `models` and update the guidelines.

Reply:
Ok, it was kinda confusing because I checked the guidelines recently and `options` was there. Anybody reading them will probably include options classes in a package like this one.

Reply:
When are we planning on starting to enforce this?